.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,24 @@
+name: Bug Report
+description: Thanks for taking the time to fill out this bug report!
```

```suggestion
description: Thanks for taking the time to fill out this bug report!
label: [bug]
```
See https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

---

Done

---

Looks good, untested. Let's push.